lib/matplotlib/tri/tritools.py

```diff
@@ -61,22 +54,21 @@ def circle_ratios(self, rescale=True):
 
         To avoid unduly low values due to a difference of scale between the 2
         axis, the triangular mesh can first be rescaled to fit inside a unit
-        square with :attr:`scale_factors` (Only if *rescale* is True, which is
```

Comment:
well, it's an :attr:, no?
(what are we doing about them, btw?)

Reply:
It's actually a property and now links there.